apiserver/pkg/model/converter.go

```diff
@@ -14,6 +15,45 @@ import (
 	v1 "k8s.io/api/core/v1"
 )
 
+func getNodeDefaultAnnotations() []string {
+	return []string{
+		"ray.io/compute-image", "openshift.io/scc", "cni.projectcalico.org/podIP", "ray.io/health-state",
+		"ray.io/ft-enabled", "cni.projectcalico.org/podIPs", "cni.projectcalico.org/containerID", "ray.io/compute-template",
+		"k8s.v1.cni.cncf.io/network-status", "k8s.v1.cni.cncf.io/networks-status",
+	}
+}
+
+func getNodeDefaultLabels() []string {
+	return []string{
+		"app.kubernetes.io/created-by", "app.kubernetes.io/name", "ray.io/cluster", "ray.io/cluster-dashboard",
+		"ray.io/group", "ray.io/identifier", "ray.io/is-ray-node", "ray.io/node-type",
+	}
+}
+
+func getHeadNodeEnv() []string {
+	return []string{
+		"MY_POD_IP", "RAY_CLUSTER_NAME", "RAY_PORT", "RAY_ADDRESS", "RAY_USAGE_STATS_KUBERAY_IN_USE",
+		"REDIS_PASSWORD",
+	}
+}
+
+func getWorkNodeEnv() []string {
+	return []string{
+		"RAY_DISABLE_DOCKER_CPU_WARNING", "TYPE", "CPU_REQUEST", "CPU_LIMITS", "MEMORY_REQUESTS",
```

Comment:
@blublinsky would you please help on list them separately in a style of below, and give a comment of this ENVs usage in this or an additional PR?
```go
const (
	// used to indicate if we need to disable docker cpu warning
	RAY_DISABLE_DOCKER_CPU_WARNING = "RAY_DISABLE_DOCKER_CPU_WARNING"
)
```